build-packages/magento-scripts/lib/config/versions/magento-2.4.2-p1.js

```diff
@@ -15,13 +14,11 @@ const { mariadb104 } = require('../services/mariadb/versions')
 const { elasticsearch712 } = require('../services/elasticsearch/versions')
 
 /**
- * @param {Object} param0
- * @param {string} param0.templateDir
- * @returns {import('../../../typings/index').CMAConfiguration & { magentoVersion: string, isDefault?: boolean }}
+ * @type {import('../../../typings/common').MagentoVersionConfigurationFunction}
  */
 module.exports = ({ templateDir }) => ({
     magentoVersion: '2.4.2-p1',
-    isDefault: true,
+    isDefault: false,
```

Comment:
should be removed from all configurations. probably should just move this in a separate variable and change it when updating magento versions